docs/concepts/gardenlet.md

```diff
@@ -1,119 +1,137 @@
 
-### Gardenlet certificate rotation
+## Gardenlet Certificate Rotation
 
-The Gardenlet tries to automatically rotate the certificate when it approaches expiration of its one year lifetime (at ~80% already passed).
-In order to use certificate rotation, the Gardenlet [component configuration](#component-configuration) needs to set the field `.gardenClientConnection.kubeconfigSecret`, specifying the secret to store the kubeconfig with the rotated certificate.
+The certificate used to authenticate the gardenlet against the API server is valid for a year. After about 10 months, the gardenlet tries to automatically replace the current certificate with a new one (certificate rotation).
 
-The same control loop in the gardener-controller-manager that signs the CSRs during the initial TLS Bootstrapping, also automatically signs the CSR during a certificate rotation.
-This works when the Gardenlet created the certificate during the initial TLS Bootstrapping using the Bootstrap kubeconfig. 
+To use certificate rotation, you need to specify the secret to store the `kubeconfig` with the rotated certificate in field `.gardenClientConnection.kubeconfigSecret` of the gardenlet [component configuration](#component-configuration).
 
-However, when trying to rotate a Kubeconfig containing a custom certificate (not created by Gardenlet TLS Bootstrap), the x509 certificate's `Subject` field needs to conform to the following:
+### Rotate certificates using bootstrap `kubeconfig`
+
+If the gardenlet created the certificate during the initial TLS Bootstrapping using the Bootstrap `kubeconfig`, certificates can be rotated automatically. 
+The same control loop in the `gardener-controller-manager` that signs the CSRs during the initial TLS Bootstrapping also automatically signs the CSR during a certificate rotation.
+
+### Rotate Certificate Using Custom `kubeconfig`
+
+When trying to rotate custom certificate that wasn’t created by gardenlet as part of the TLS Bootstrap, the x509 certificate's `Subject` field needs to conform to the following:
   - the Common Name (CN) is prefixed with `gardener.cloud:system:seed:`
-  - the Organisation (O) equals `gardener.cloud:system:seeds`
+  - the Organization (O) equals `gardener.cloud:system:seeds`
+
+Otherwise, the `gardener-controller-manager` doesn’t automatically sign the CSR.
+In this case, an external component or user needs to approve the CSR manually, for example, using command  `kubectl certificate approve  seed-csr-<...>`).
+If that doesn’t happen within 15 minutes, the gardenlet repeats the process and creates another CSR.
 
-Otherwise, the gardener-controller-manager will not automatically sign the CSR.
-In this case, an external component/user needs to approve the CSR manually (e.g. via `kubectl certificate approve  seed-csr-<...>`).
-If that does not happen within 15 minutes, the Gardenlet repeats the process and creates another CSR.
+## Seed Config versus Seed Selector
```

Comment:
```suggestion
## Seed Config vs. Seed Selector
```

Reply:
Is there an essential reason to abbreviate "versus" to "vs." here? The title is not very long, and abbreviations negatively affect readability and comprehensibility. They increase cognitive load because you have to remember what they stand for.

Reply:
>  Is there an essential reason to abbreviate "versus" to "vs." here?

Well, it was "vs." on purpose to highlight the two different seed options (config / selector), that's why I suggested to simply keep it like this. 

> and abbreviations negatively affect readability and comprehensibility. They increase cognitive load because you have to remember what they stand for.

I don't agree to such general statement. There are a lot of well-established abbreviations like "vs.", "i.e.", "e.g.", that will not any cognitive overload for anybody. It's rather the opposite, NOT using the abbreviation will cause this "load" because you barely see it anywhere.

Anyways, I don't insist on that one/care too much about it, you are the expert, but while reading it I stumbled across it and felt "vs." is way cleaner.